src/main/java/com/google/javascript/gents/ModuleConversionPass.java

```diff
@@ -300,6 +300,12 @@ public void visit(NodeTraversal t, Node n, Node parent) {
         }
       }
     }
+
+    private boolean isARequireLikeCall(Node node) {
```

Comment:
Should this check that node itself is a call node or something?

Reply:
done, also cleaned up the destructuring call.